main.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'app'
+
+def print_menu
+  puts 'Please choose an option:'
+  puts '1. List all books'
+  puts '2. Add a book'
+  puts '3. Rent a book'
+  puts '4. Return a book'
+  puts '5. List all rentals'
+  puts '6. Quit'
```

Comment:
- [ ] Let's ensure these options here match the ones given in this [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). The options should be as shown below. Kindly implement them and test to make sure they work as expected.

![Screenshot 2023-02-17 at 09 14 02 (2)](https://user-images.githubusercontent.com/64774936/219593226-1a4a7b9f-65c6-443b-8403-09a25270887a.png)